ccm_web/server/src/api/api.admin.handler.ts

```diff
@@ -38,10 +39,12 @@ or account-scoped operations that make use of other handler instances for Canvas
 export class AdminApiHandler {
   requestor: CanvasRequestor
   userLoginId: string
+  maxSearchCourses: number
 
-  constructor (requestor: CanvasRequestor, userLoginId?: string) {
+  constructor (requestor: CanvasRequestor, userLoginId?: string, maxSearchCourses = 400) {
```

Comment:
What do we need to set this number here instead of getting it from the config?

Reply:
So we are setting it in the config, and passing it where it is needed. This is a concession to the fact that the `AdminApiHandler` is also used by the `createExternalUser` endpoint, which doesn't need `maxSearchCourses` (or `userLoginId` for that matter). It doesn't seem like we should require that endpoint to specify these if it's not going to use them, but also not setting them in the constructor has implications for `undefined`s in the course search process.

Ideally speaking, I'd probably lean towards pulling out the `createExternalUser` and `getUserInfo` calls into a separate class so can avoid this awkwardness.

Reply:
I documented by thinking here: https://github.com/tl-its-umich-edu/canvas-course-manager-next/issues/380
Icebox is appropriate.